yield_plugin/include/yield_plugin/yield_plugin.hpp

```diff
@@ -60,6 +60,16 @@ struct PointSpeedPair
   double speed = 0;
 };
 
+/**
+ * \brief Convenience class for saving collision results
+ */
+struct CollisionData
```

Comment:
Consider renaming this to `GetCollisionResult` to better align with the Standard Library's `*_result` types. This kind of struct, coupled with its usage, is intended to facilitate multiple return values from `get_collision()`. It's not really something to pass around to other functions.

Reply:
Renamed